src/components/PrivacyTerms.js

```diff
@@ -0,0 +1,57 @@
+import { h } from 'preact'
+import { useRef, useEffect } from 'preact/hooks'
+import { useNavigation, useI18n, useSoftkey, usePopup } from 'hooks'
+import { ListView, UsageReportConsent } from 'components'
+import { getAppLanguage } from 'utils'
+
+export const PrivacyTerms = ({ close }) => {
+  const containerRef = useRef()
+  const listRef = useRef()
+  const i18n = useI18n()
+  const lang = getAppLanguage()
+  const [showUsageReportConsent] = usePopup(UsageReportConsent, { mode: 'fullscreen', stack: true })
+
+  const onKeyCenter = () => {
+    const { index } = getCurrent()
+    const item = items[index]
+
+    // open link
+    if (item.link) {
+      window.open(item.link)
+    } else if (item.action) {
+      item.action()
+    }
+  }
+
+  const onUsageReportConsentSelected = () => {
+    showUsageReportConsent()
+  }
+
+  useSoftkey('PrivacyTerms', {
+    left: i18n('softkey-close'),
+    onKeyLeft: close,
+    center: i18n('centerkey-select'),
+    onKeyCenter,
+    onKeyBackspace: close
+  })
+
+  const [, setNavigation, getCurrent] = useNavigation('PrivacyTerms', containerRef, listRef, 'y')
+
+  useEffect(() => {
+    setNavigation(0)
+  }, [])
+
+  const items = [
+    { title: i18n('settings-usage-reports'), action: onUsageReportConsentSelected },
+    { title: i18n('settings-privacy'), link: 'https://foundation.m.wikimedia.org/wiki/Privacy_policy' },
```

Comment:
The privacy policy URL could use the `/${lang}` as well.

Reply:
Hmm yea interesting point, what I'm seeing is:

- `https://foundation.m.wikimedia.org/wiki/Privacy_policy` has a map for other privacy policy in other languages
- looks like the privacy policy page in english is `https://foundation.m.wikimedia.org/wiki/Privacy_policy`
  - because `https://foundation.m.wikimedia.org/wiki/Privacy_policy/en` lands in a page with no text
  - same with non-mobile version `https://foundation.wikimedia.org/wiki/Privacy_policy/en`
- seems like other languages are directed to `meta.wikimedia`: `https://meta.wikimedia.org/wiki/Privacy_policy/fr`
- and `https://meta.wikimedia.org/wiki/Privacy_policy/en` actually lands on a page with what seems to be the same content as `https://foundation.m.wikimedia.org/wiki/Privacy_policy`

Should we just simply update the link `https://meta.m.wikimedia.org/wiki/Privacy_policy/${lang}`. This would deviate from the link specified in the Phabricator ticket description, but we would be saving the user the step of choosing their language

Reply:
Nevermind, let's keep linking to foundation wiki. This is a legal document so we need to be sure where we link to. Foundation wiki is controlled by the foundation but meta wiki is publicly editable.